src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -12,23 +12,43 @@
 namespace Symfony\Component\Cache\Adapter;
 
 use Psr\Cache\CacheItemInterface;
-use Psr\Cache\CacheItemPoolInterface;
 use Symfony\Component\Cache\CacheItem;
 use Symfony\Component\Cache\Exception\InvalidArgumentException;
 
 /**
  * @author Nicolas Grekas <p@tchwork.com>
  */
-abstract class AbstractAdapter implements CacheItemPoolInterface
+abstract class AbstractAdapter implements AdapterInterface
 {
+    /**
+     * @var string
+     */
     private $namespace;
+
+    /**
+     * @var AdapterInterface
+     */
+    private $decorated;
+
+    /**
+     * @var array
```

Comment:
This PR has been replaced by https://github.com/symfony/symfony/pull/17556. The new one doesn't include docblocks changes anymore.